### ISSUE TITLE
Easier access to the preset pass managers

### ISSUE THREAD

**Issue by kdk** [Qiskit/qiskit-terra#7023]
### What is the expected enhancement?

Currently, accessing the preset passmanagers (outside of a call to `transpile`) requires building a `pass_manager_config` by hand, like:

```
from qiskit.transpiler import PassManagerConfig, CouplingMap
from qiskit.transpiler.preset_passmanagerrs import *

pass_manager_config = qk.transpiler.PassManagerConfig(
    basis_gates = backend.configuration().basis_gates,
    coupling_map = CouplingMap(backend.configuration().coupling_map),
    backend_properties = backend.properties(),
)

pm = level_3_pass_manager(pass_manager_config)
```

which is a fair amount of boilerplate if one already has access to a backend and a set of transpiler options (and prone to going stale as new options are added). I also don't think we document this anywhere.

One option would be to add a `PassManagerConfig.from_backend(backend, **pass_manager_options)` so that users can `level_1_pass_manager(PassManagerConfig.from_backend(backend, **pass_manager_options))` and we can extract some of the argument parsing and backend extraction logic from `qiskit.compiler.transpiler`.

Another would be to build upon the builder for the preset pass managers being introduced in #6403 and have something like `pm = FullPassManager.from_backend(backend, **pass_manager_options)`

(Related to #3118 .)

**Comment by HuangJunye** [comment]
Should this issue be closed since it's fixed by #7163?

**Comment by kdk** [comment]
Closing as resolved by #7163 .
